packages/table/src/quadrants/_quadrants.scss

```diff
@@ -94,8 +94,10 @@ $table-quadrant-scroll-container-overflow: 20px;
 
   .bp-table-quadrant-scroll-container {
     position: absolute;
+    top: 0;
     right: -$table-quadrant-scroll-container-overflow;
-    height: 100%;
+    bottom: 0;
+    height: auto;
```

Comment:
Whenever I specify both `top` and `bottom`, I like to take `height` out of the picture so the two schemes don't compete unpredictably.